packages/cubejs-docker/README.md

```diff
@@ -1,25 +1,100 @@
 <p align="center"><a href="https://cube.dev"><img src="https://i.imgur.com/zYHXm4o.png" alt="Cube.js" width="300px"></a></p>
 
-[Website](https://cube.dev) • [Docs](https://cube.dev/docs) • [Blog](https://cube.dev/blog) • [Slack](https://slack.cube.dev) • [Twitter](https://twitter.com/thecubejs)
+[Website](https://cube.dev) • [Docs](https://cube.dev/docs) •
+[Blog](https://cube.dev/blog) • [Slack](https://slack.cube.dev) •
+[Twitter](https://twitter.com/thecubejs)
 
 # Supported tags and respective `Dockerfile` links
 
-- [latest](https://github.com/cube-js/cube.js/blob/master/packages/cubejs-docker/latest.Dockerfile) - Latest stable release based on Debian (recommended)
-- [alpine](https://github.com/cube-js/cube.js/blob/master/packages/cubejs-docker/latest-alpine.Dockerfile) - Latest stable release on Alpine
-- [dev](https://github.com/cube-js/cube.js/blob/master/packages/cubejs-docker/dev.Dockerfile) - Latest development release from master branch.
+- [latest](https://github.com/cube-js/cube.js/blob/master/packages/cubejs-docker/latest.Dockerfile) -
+  Latest stable release, based on Debian (recommended)
+- [alpine](https://github.com/cube-js/cube.js/blob/master/packages/cubejs-docker/latest-alpine.Dockerfile) -
+  Latest stable release on Alpine
+- [dev](https://github.com/cube-js/cube.js/blob/master/packages/cubejs-docker/dev.Dockerfile) -
+  Latest development release from master branch.
+
+# Quick reference
+
+Where to get help? Check out our Slack, Stack Overflow or GitHub Discussions
+pages.
+
+Found an issue? File it on our GitHub Issues page.
+
+Supported architectures: `amd64`
```

Comment:
Only AMD64?

Reply:
I think as of today, yes. 

@cube-js/core can you confirm please?

Reply:
Yes, probably We will support ARM64 in the feature - https://github.com/cube-js/cube.js/pull/1388